BepInEx.Core/Configuration/ConfigFile.cs

```diff
@@ -93,6 +93,11 @@ public ConfigFile(string configPath, bool saveOnInit, BepInPlugin ownerMetadata)
 		#region Save/Load
 
 		private readonly object _ioLock = new();
+		
+		/// <summary>
+		/// Generate user-readable comments for each of the settings in the saved .cfg file.
+		/// </summary>
+		public bool GenerateSettingDescriptions { get; set; } = true;
```

Comment:
This may change down the road depending on where we will take the config system, but for this whis is fine.